addons/istio/1.5.x/istio-1.yaml

```diff
@@ -34,61 +33,7 @@ spec:
       enabled: false
     - name: none
       enabled: false
-  chartReference:
-    chart: istio
-    repo: https://mesosphere.github.io/charts/staging
-    version: 1.4.3
-    values: |
-      kiali:
-       enabled: true
-       contextPath: /ops/portal/kiali
-       ingress:
-         enabled: true
-         kubernetes.io/ingress.class: traefik
-         hosts:
-           - ""
-       dashboard:
-         auth:
-           strategy: anonymous
-       prometheusAddr: http://prometheus-kubeaddons-prom-prometheus.kubeaddons:9090
-
-      tracing:
-        enabled: true
-        contextPath: /ops/portal/jaeger
-        ingress:
-          enabled: true
-          kubernetes.io/ingress.class: traefik
-          hosts:
-            - ""
-
-      grafana:
-        enabled: true
-
-      prometheus:
-        serviceName: prometheus-kubeaddons-prom-prometheus.kubeaddons
-
-      istiocoredns:
-        enabled: true
-
-      security:
-       selfSigned: false
-       caCert: /etc/cacerts/tls.crt
-       caKey: /etc/cacerts/tls.key
-       rootCert: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-       certChain: /etc/cacerts/tls.crt
-       enableNamespacesByDefault: false
-
-      global:
-       podDNSSearchNamespaces:
-       - global
-       - "{{ valueOrDefault .DeploymentMeta.Namespace \"default\" }}.global"
-       defaultPodDisruptionBudget:
-        enabled: false
-
-       mtls:
-        enabled: true
-
-       multiCluster:
-        enabled: true
-
-       controlPlaneSecurityEnabled: true
+  kudoReference:
```

Comment:
What happens with an existing Addon in this case where we change the deployment machinery from Helm to KUDO? Will the Helm release be deleted properly by kubeaddons?

Reply:
Istio is still in a preview with no upgrade support